src/server.rs

```diff
@@ -28,10 +28,17 @@ use crate::jobserver::Client;
 use crate::mock_command::{CommandCreatorSync, ProcessCommandCreator};
 use crate::protocol::{Compile, CompileFinished, CompileResponse, Request, Response};
 use crate::util;
+use bytes_05::{Bytes, BytesMut};
 use filetime::FileTime;
-use futures::sync::mpsc;
-use futures::{future, stream, Async, AsyncSink, Future, Poll, Sink, StartSend, Stream};
-use futures_03::compat::Compat;
+use futures::Future as _;
+use futures_03::{
+    channel::mpsc,
+    compat::*,
+    future,
+    prelude::*,
+    ready, stream,
+    task::{self, Poll},
```

Comment:
`Poll` is a reexport of `std::task::Poll`, please import `Poll` from there instead. As a follow-up, we can replace `std::task::Poll` with `Poll` in this file.